cdap-app-templates/cdap-etl/hydrator-spark-core-base/src/main/java/io/cdap/cdap/etl/spark/streaming/DStreamCollection.java

```diff
@@ -78,9 +79,12 @@ public DStreamCollection(JavaSparkExecutionContext sec, JavaDStream<T> stream) {
 
   @Override
   public SparkCollection<T> cache() {
-    SparkConf sparkconf = stream.context().sparkContext().getConf();
-    if (sparkconf.getBoolean(Constants.SPARK_PIPELINE_AUTOCACHE_ENABLE_FLAG, true)) {
-      return wrap(stream.cache());
+    SparkConf sparkConf = stream.context().sparkContext().getConf();
+    if (sparkConf.getBoolean(Constants.SPARK_PIPELINE_AUTOCACHE_ENABLE_FLAG, true)) {
+      String cacheStorageLevelString = sparkConf.get(Constants.SPARK_PIPELINE_CACHING_STORAGE_LEVEL, 
+          Constants.DEFAUL_CACHING_STORAGE_LEVEL);
```

Comment:
nit: use indentation consistent with rest of project. In this situation, should line up with the argument from the previous line.